### ISSUE TITLE
Controlling multiple same air conditioners independently

### ISSUE THREAD

**Issue by hariskhalid** [tpudlik/RaspAC#2]
Hi
Thanks for your answer I am now able to control my air conditioner from the raspberry pi. Now that the that part is done i want to control other air conditioners too. I have 2 Ac’s of the same company(same model) that i want to control independently from the raspberry pi. Is it possible that i can do this

**Comment by tpudlik** [comment]
I believe it's not possible to achieve this through software: both ACs will respond to any signal they receive.  You would need two (sets of) IR diodes that you control separately, set up so that each AC is reached by light from only one set.

**Comment by hariskhalid** [comment]
YES yes i am using two IR diodes but the problem is I think that you can control only 1 gpio pin using lirc at a time. Is it correct?

**Comment by tpudlik** [comment]
That's a good point, I forgot that only the hardware PWM pin can be used with lirc .  In this case, I'm afraid you would have to build some more circuitry (a transistor switch controlled by some other pin) to divide this pin's signal between two wires.
